### DISCUSSION TITLE
[Changelog] Apply Common Community Standard

### DISCUSSION THREAD

**Discussion by kevinmatthes** [zee-editor/zee#56] [Show and tell]
I would like to suggest to format the changelog by applying a community standard therefore.  This thread is intended to discuss the possible standards as well as their pros and cons.

Applying any form of convention at all is an enhancement in every case since people visiting this repository will then be able to navigate through the changelog more easily.

**Comment by kevinmatthes** [comment]
Keep a Changelog

**Reply by kevinmatthes** [reply]
It is well-readable and lightweight when applying.  As an example for a repository which already applies this convention, I would like to name [`sysexits`](https://github.com/sorairolake/sysexits-rs), a Rust crate for application exit codes.

**Reply by iainh** [reply]
Looking at the format that @mcobzarenco  has in the existing Changelog.md, it looks like a format matching Keep a Changelog has been implemented in https://github.com/zee-editor/zee/commit/0b4078470ff6be6cb60388189d08531c368d2cf4. Does that match what you were thinking or do you think there is more we could do in this document?

**Reply by kevinmatthes** [reply]
I agree that the basic style how the current CHANGELOG is written is close to the requirements of Keep a Changelog but I would like to suggest to match the category naming and the release date annotation.  This would simplify parsing the file, for instance to create a summary for a GitHub release.

**Reply by mcobzarenco** [reply]
Happy to adopt Keep a Changelog as the house style. As @iainh notes it's pretty much there.

**Reply by mcobzarenco** [reply]
I created a PR https://github.com/zee-editor/zee/pull/64
